src/main/java/com/prgrms/springbootbasic/voucher/domain/FixedAmountVoucher.java

```diff
@@ -27,9 +32,24 @@ public FixedAmountVoucher(UUID uuid, int fixedAmount) {
         this.fixedAmount = fixedAmount;
     }
 
+    public FixedAmountVoucher(UUID uuid, int fixedAmount, VoucherType voucherType) {
+        validate(fixedAmount);
+        this.id = uuid;
+        this.fixedAmount = fixedAmount;
+        this.voucherType = voucherType;
+    }
+
+    public FixedAmountVoucher(int fixedAmount, VoucherType voucherType) {
+        validate(fixedAmount);
+        this.id = UUID.randomUUID();
+        this.fixedAmount = fixedAmount;
+        this.voucherType = voucherType;
+    }
```

Comment:
```suggestion
    public FixedAmountVoucher(int fixedAmount, VoucherType voucherType) {
        this(null, fixedAmount....
    }
```
생성자 오버로딩을 해보면 좋을 것 같아요. 동일한 비즈니스를 타야 validate같은 메서드가 누락되는걸 방지할 수 있습니다.